controllers/user.controller.js

```diff
@@ -1,75 +1,83 @@
 const httpStatus = require('http-status');
+
 const User = require('../models/user.model');
 
 const createUser = async (req, res) => {
   try {
-    // tạo mới user đã tồn tại email
-    // password => hash (npm i brcrypt)
-    // password không được trả về kèm response
-    const { fullname, email, password } = req.body;
-    if (!fullname || !email || !password) {
+    const {fullname, email, password} = req.body;
+    if(!fullname || !email || !password){
       return res.status(httpStatus.BAD_REQUEST).json({
-        message: 'Vui lòng điền đầy đủ thông tin',
+        message: 'Vui lòng nhập đầy đủ thông tin',
         code: httpStatus.BAD_REQUEST,
       });
     }
-    const user = await User.create({ fullname, email, password });
+    const users = await User.create({fullname, email, password});
```

Comment:
```suggestion
    const user = await User.create({fullname, email, password});
```
hàm này đang xử lý tạo một người dùng nên biến trả về định nghĩa số ít `user`